### PR TITLE
Don't prompt to choose installation type when user has provided database options

### PR DESCRIPTION
**Pull request by cupofnestor**
### What does it do?

Allows user to bypass interactive prompts completely, by passing database arguments via cli.

### Why is it needed?

This allows completely automated installation of strapi via ansible/scripting or similar.

### How to test it?

Initially, using node and postgresql, ran the command:
```bash
  node /path/to/my/strapi-fork/strapi/packages/cli/create-strapi-app/create-strapi-app.js ./ \
  --dbclient=postgres \
  --dbhost=127.0.0.1 \
  --dbport=5432 \
  --dbname=strapi \
  --dbusername=strapi \
  --dbpassword=strapi \
```

It also seemed appropriate to run the e2e test:
`node test/e2e.js`

### Related issue(s)/PR(s)

none

### CODE REVIEW

**Review by alexandrebodin — COMMENTED** [review]
Thank you that's a nice improvement we had that behavior before it's much better I think

File: packages/cli/create-strapi-app/create-strapi-app.js [hunk 1]
```diff
@@ -80,8 +80,8 @@ async function initProject(projectName, program) {
     return generateApp(projectName, program);
   }
 
-  const prompt = await promptUser(projectName, program);
-
+  const prompt = await promptUser(projectName, program, hasDatabaseOptions);
+  console.log(prompt);
```
[the remark this answers is not in the document]
Comment (cupofnestor) [reply packages/cli/create-strapi-app/create-strapi-app.js:84]: you got me

**Comment by alexandrebodin** [comment]
You have a few conflict in yarn you will have to resolve so the PR passes the checks :)

**Comment by cupofnestor** [comment]
@alexandrebodin anything else I need to do here?  I think I've resolved the conflict in yarn lock.

**Review by alexandrebodin — APPROVED** [review]
LGTM 👍

**Pull request merged**